src/components/Calendar/Weekdays.tsx

```diff
@@ -9,19 +10,28 @@ export const FORMAT = 'DD/MM/YYYY';
 export interface Props {
   week?: dayjs.Dayjs[];
   onSelect?: (selected: dayjs.Dayjs) => void;
-  selected?: dayjs.Dayjs | null;
+  selected: dayjs.Dayjs[];
   state?: any;
 }
 
 function Weekdays({ week = [], onSelect = () => {}, selected, state = {} }: Props) {
+  dayjs.extend(isBetween);
```

Comment:
you should not do this inside of the component. you can move this to the top of the file after the import.